protocol/v2/ssv/msgqueue/signed_msg.go

```diff
@@ -73,7 +72,7 @@ func SignedMsgIndex(msgType spectypes.MsgType, mid string, h specqbft.Height, cm
 // TODO: identify decided messages by: 1) type commit; 2) quorum of signers
 func DecidedMsgIndexer() Indexer {
 	return func(msg *spectypes.SSVMessage) Index {
-		if msg.MsgType != message.SSVDecidedMsgType && msg.MsgType != spectypes.SSVConsensusMsgType {
+		if msg.MsgType != spectypes.SSVConsensusMsgType {
```

Comment:
add k.D check

Reply:
there's no `k.D` here, the `msg.MsgType != spectypes.SSVConsensusMsgType` check covers decided messages as well